doc/nrf/known_issues.rst

```diff
@@ -2212,6 +2212,11 @@ SoftDevice Controller
 
 In addition to the known issues listed here, see also :ref:`softdevice_controller_limitations` for permanent limitations.
 
+.. rst-class:: v2-2-0 v2-1-3 v2-1-2 v2-1-1 v2-1-0
+
+DRGN-18568: Lower radio output power when :kconfig:option:`BT_CTLR_TX_PWR_ANTENNA` and :kconfig:option:`MPSL_FEM` is used.
```

Comment:
This also applies to CONFIG_BT_CTLR_TX_PWR so maybe this statement should be generalized a bit. Something like "when setting a default TX power and :kconfig:option:`MPSL_FEM` is used.". Or both kconfigs could be mentioned.